app-creator/src/redux/actions.ts

```diff
@@ -114,3 +185,25 @@ export const incrementWidgetID = (id: string): IncrementWidgetAction => {
     },
   };
 };
+
+/**
+ * Returns default values for a specified widget type (ie. label, button, etc).
+ */
+function getDefaultUniqueAttributes(type: string): UniqueAttributes {
```

Comment:
Can we get a consistent naming for the function and the return type?
e.g. `getDefaultAttributes(type: string): DefaultAttributes`, or 
`getUniqueAttributes(type: string): UniqueAttributes`